### DISCUSSION TITLE
Discord Time

### DISCUSSION THREAD

**Discussion by uberlinuxguy** [OpenParsec/openparsec#60] [Announcements]
Yeah, I've decided to make a discord for OpenParsec.

Here's the link: https://discord.gg/G7ZejM5N5h

@mattthias this is your fault... :-)

**Comment by Cheeseness** [comment]
Don't want to be discouraging, but what was wrong with the IRC channel listed on the openparsec.com contact page? It's not super active, but there are people in there ( @mattthias and I were chatting in there before the recent issues were created ) :D

**Comment by uberlinuxguy** [comment]
Oh, I mean, yeah, I didn't know that was still around.... we can use that too..... 
I can update the README.md with that.

**Comment by uberlinuxguy** [comment]
Or I should say I can use that.... I've been using discord a lot lately.... forgot about IRC... which means I have to give up my old fart card.

**Comment by uberlinuxguy** [comment]
Just for funsies, was there a good web based IRC client anyone uses?

**Comment by Cheeseness** [comment]
Following discussion in IRC, mibbit looks like it could be a appropriate - maybe it'd be worth embedding that on the website to make the channel more accessible to anybody less familiar with IRC.

**Comment by CrazySpence** [comment]
The IRC channel has no owner cause it expired some time ago, Discord is far more acceptable to use these days. I'm only still on slashnet cause I can't be bothered to take the time to remove it from my ZNC profile

**Comment by uberlinuxguy** [comment]
Gotta watch my git accounts.... mixing and such...

Anyway, as I was saying, I talked to one of the IRC server gods today, and they granted @CrazySpence op on the openparsec IRC channel.

I'm fine camping in both places, personally, but it might be a good idea to pick one if we are going to start playing around again..
